pkg/systemtemplate/template.go

```diff
@@ -197,4 +197,109 @@ spec:
           secretName: cattle-credentials-{{.TokenKey}}
   updateStrategy:
     type: RollingUpdate
+---
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
How are these CRDs related or not realted to the ones you create via BatchCreateCRDs in clusterdeploy.go